pom.xml

```diff
@@ -580,8 +580,8 @@
         <commons-io.wso2.version.range>[2.4.0, 3.0.0)</commons-io.wso2.version.range>
         <org.apache.commons.commons-lang3.version>3.1</org.apache.commons.commons-lang3.version>
         <org.apache.commons.commons-lang3.version.range>[3.1, 4.0)</org.apache.commons.commons-lang3.version.range>
-        <guava.version>18.0</guava.version>
-        <guava.version.range>[18.0,19.0)</guava.version.range>
+        <guava.version>21.0</guava.version>
+        <guava.version.range>[20.0,30.0)</guava.version.range>
```

Comment:
IMO version range should be `[21.0,22.0)` as the next major release after version 21.0 is [22.0](https://github.com/google/guava/releases/tag/v22.0-rc1).